src/ios/CodePush.m

```diff
@@ -235,7 +235,9 @@ - (BOOL)loadPackage:(NSString*)packageLocation {
 }
 
 - (void)loadURL:(NSURL*)url {
-    [((UIWebView*)((CDVViewController *)self.viewController).webView) loadRequest:[NSURLRequest requestWithURL:url]];
+    // Cast to a UIWebView here to enable call to loadRequest.
+    // Both UIWebView and WKWebview share this call, so regardless of which type the returned webview is, this will work.
+    [(UIWebView*)self.webView loadRequest:[NSURLRequest requestWithURL:url]];
```

Comment:
Ah. I don't think i tested with 3.9.0. I'll hit that today and see if this works.